src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int RADIUS = 10;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figureType = random.nextInt(4); // 4 different type of figures
```

Comment:
According to the checklist, all magic numbers in your code should be constants. You should define a constant for the number of figure types. For example, `private static final int FIGURE_TYPES = 4;` and then use `random.nextInt(FIGURE_TYPES);`.